test-flip-link-app/examples/crash.rs

```diff
@@ -4,6 +4,7 @@
 use core::ptr;
 
 use cortex_m_rt::entry;
+#[cfg(feature = "default")]
```

Comment:
thinking more about this: could you use `#[cfg(feature = "lm3s6965")]` here and elsewhere the "default" feature is currently used? I think that "default" doesn't clearly convey what, or which, feature(s) are being checked -- the "default" feature could be enabling several Cargo features (I know it's not in this case).